registration/templates/registration/terms.html

```diff
@@ -0,0 +1,514 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block title %}Terms of Use{% endblock title %}
+
+{% block content %}
+<div class="page-title text-xs-center color-white">
+    <h1 class="page-title__text">Terms of Use</h1>
+</div>
+
+<div class="row page-content clearfix">
+    <div class="page-content__inner small-12 medium-10 medium-offset-1 float-left background-white">
+        <div class="page-content__inner__wrapper">
+            <section>
+                <h3>General</h3>
+                <p>
+                    Headquarters for the National Digital Israel Initiative in
```

Comment:
@jamestait, is this the correct ToS? If not, can we add the correct text?

Reply:
@lgp171188, I believe so. The document wasn't linked from the ticket, so I went searching for it. @viadanna confirmed I'd got the correct documents, and was looking for an email confirmation to make doubly sure.

Reply:
@jamestait, this mentions some organization in Israel and has nothing to do with it. @viadanna

Reply:
@jamestait There's been some confusion, I guess this might be a sample document. The correct ones are linked in the ticket.

Reply:
OK, I'll get that updated. Thanks for the corrected links. :slightly_smiling_face:

Reply:
Pushed the updated ToS.